docs/_docs/persistence/change-data-capture.adoc

```diff
@@ -142,7 +142,26 @@ IMPORTANT: `ignite-cdc.sh` implements the fail-fast approach. It just fails in c
  5. Infinitely wait for the newly available segment and process it.
  6. Stop the consumer in case of a failure or a received stop signal.
 
+== Handling skipped segments
+
+The CDC can be disabled manually or by configured directory maximum size. In this case a hard link creation will be skipped.
```

Comment:
So when enabled there will be gap between segments: `0000000000000002.bin, 0000000000000010.bin, 0000000000000011.bin`, for example.

Reply:
Fixed